modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -40,331 +40,52 @@
       it 'matches the expected schema' do
         # TODO: this should use the matcher helper instead (was throwing an Oj::ParseError)
         # expect().to match_json_schema('immunizations')
-        expect(response.parsed_body).to eq(
```

Comment:
Given that this is just supposed to be a schema test, I narrowed it down to a single record for readability.